recipes/centos_spring_boot/Dockerfile

```diff
@@ -16,7 +16,10 @@ ARG JUNIT_VERSION=4.12
 ENV SPRING_BOOT_GROUP=org.springframework.boot
 
 COPY install_spring_boot_dependencies.sh /tmp/
-RUN sudo chown user:user /tmp/install_spring_boot_dependencies.sh && \
-    sudo chmod a+x /tmp/install_spring_boot_dependencies.sh && \
+COPY ./run.sh ${HOME}/
```

Comment:
question: do we really need to leave the command in ${HOME} folder ?
user may want to run that script as well in the terminal ?